src/legacy/blueprint.coffee

```diff
@@ -218,6 +218,8 @@ class Resource
       resolvedAttributes:  undefined
       actionAttributes:    undefined
       resolvedActionAttributes: undefined
+      actionRelation:      undefined
```

Comment:
@klokane you need to add this to `toJSON` and `fromJSON` in order for serialization to work, pity it is not tested ...